src/style.css

```
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--1 .stars_star:nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars_star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}

```

```suggestion
.stars--1 .stars_star:nth-child(1),
.stars--2 .stars_star:nth-child(-n+2) {
  background-image: url(./images/star-active.svg);
}

```
you can group all this cases like this